### DISCUSSION TITLE
İn claim single player lock system.

### DISCUSSION THREAD

**Discussion by successed** [GriefPrevention/GriefPrevention#626] [Ideas]
The Griefprevention plugin does not allow players in the same section to lock their chests. So if you are registered to a claim, other players in that claim can open your chests. How can I prevent this?

**Comment by RoboMWM** [comment]
what?
Unless you trusted players, they can't open your chests in your claims...

**Comment by successed** [comment]
> what?
> Unless you trusted players, they can't open your chests in your claims...

i need I want players attached to claim to lock their chests.Other trusted players must have authority to open this chest not else.

Like this;
![Screenshot_7](https://user-images.githubusercontent.com/13768414/65366213-a3b79c80-dc29-11e9-81d5-143767abdd89.png)

**Comment by RoboMWM** [comment]
Why do you need to lock chests? Chests inside of claims are already protected...

**Comment by s1h4d0w** [comment]
> 
> 
> > what?
> > Unless you trusted players, they can't open your chests in your claims...
> 
> i need I want players attached to claim to lock their chests.Other trusted players must have authority to open this chest not else.

Why not get one of the many other chest lock plugins?

**Comment by TomLewis** [comment]
I disabled the GP "lock chests" feature in favour of users using a block on top of their claimed land, if you claim something and then put a solid block on the chest it will stop the chest opening.
This means people can accidental leave them unlocked and get looted.

**Comment by successed** [comment]
> > > what?
> > > Unless you trusted players, they can't open your chests in your claims...
> > 
> > 
> > i need I want players attached to claim to lock their chests.Other trusted players must have authority to open this chest not else.
> 
> Why not get one of the many other chest lock plugins?

other players in that claim can open your chests. i need single lock system like blocklocker.

**Comment by RoboMWM** [comment]
> other players in that claim can open your chests.

Why are they trusted to that claim if you can't trust them with chests...?

**Comment by successed** [comment]
> > other players in that claim can open your chests.
> 
> Why are they trusted to that claim if you can't trust them with chests...?

In general, the problem is: You trick me by pretending to be my friend, I put you in my house, and you rob my house and run away.

**Comment by RoboMWM** [comment]
Can't they destroy the whole house too...?

**Comment by successed** [comment]
> Can't they destroy the whole house too...?

They usually put their important things in the crates.

**Comment by s1h4d0w** [comment]
> 
> 
> > Can't they destroy the whole house too...?
> 
> They usually put their important things in the crates.

There are a lot of chest lock plugins that will prevent any other player from destroying a protected chest, even if they have building rights. This seems like the best option for this.

Unless RoboMWM wants to add a "deny" option to containertrust, that overrides accessing/destroying containers for a player that has build trust. Tho that could be complicated with all the ways a player could destroy a container, fire, lava, tnt, etc. A separate plugin seems much simpler.

**Comment by successed** [comment]
> > > Can't they destroy the whole house too...?
> > 
> > 
> > They usually put their important things in the crates.
> 
> There are a lot of chest lock plugins that will prevent any other player from destroying a protected chest, even if they have building rights. This seems like the best option for this.
> 
> Unless RoboMWM wants to add a "deny" option to containertrust, that overrides accessing/destroying containers for a player that has build trust. Tho that could be complicated with all the ways a player could destroy a container, fire, lava, tnt, etc. A separate plugin seems much simpler.

Another plugin does not support griefprevention plugin.

Check this out :
Plugin : https://www.spigotmc.org/resources/blocklocker.3268/
My issue github : https://github.com/rutgerkok/BlockLocker/issues/36

**Comment by RoboMWM** [comment]
Idk how that guy even interpreted your request, but if his interpretation is correct, I agree with it; there's no need for it to "hook into" GP.

**Comment by s1h4d0w** [comment]
> 
> 
> Idk how that guy even interpreted your request, but if his interpretation is correct, I agree with it; there's no need for it to "hook into" GP.

I think he literally means, if he does `/trust` then that player can access his chests. And that's just how GP works. He was suggesting a way to do `/trust` but still have his containers locked for that player. There's already a load of plugins that do exactly that and would work just fine in combination with GP.

If you have some time and if I'm not being too forward, could you take a look at the issue I posted about some players being unable to claim in certain regions? There have been a few new players who are experiencing the issue, if you could be able to tell me how to troubleshoot this further or if this is an issue with WorldGuard (as I'm running 1.14.4) that would be awesome too.
